src/core/sockets.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import humps from 'humps';
+
+import { onWithStore } from '../sockets';
+import * as actions from './actions';
+
+export default onWithStore((store, on) => {
+  return on(
+    'epl_dashboard_EPL',
+    {
+      'system-init': data => {
+        const node = humps.camelizeKeys(data).nodeName;
+        store.dispatch(actions.setNode(node));
+      }
+    },
+    () => store.dispatch(actions.connectionOpen()),
+    () => store.dispatch(actions.connectionClose())
+  );
+});
```

Comment:
@arkgil I've added new function which allow to pass store to handler. It's due to circular dependencies which are messing imports a little, so right now store is passed as argument.